internal/cli/check-registry/check-registry.go

```diff
@@ -0,0 +1,94 @@
+// This file is part of libraries-repository-engine.
```

Comment:
Did you intentionally deviate from the established structure where the commands are implemented by packages under `internal/command`:

https://github.com/arduino/libraries-repository-engine/tree/cleanups/internal/command

It probably makes sense to put the package here if your plan is to eventually move the packages for the other commands as well, but otherwise I would suggest moving `github.com/arduino/libraries-repository-engine/internal/cli/check-registry` to `github.com/arduino/libraries-repository-engine/internal/command/check-registry`.

I don't have a strong preference about which structure is used; only that there be consistency in the organization of the packages.

Reply:
> Did you intentionally deviate from the established structure where the commands are implemented by packages under internal/command

🤦🏼 nope my mistake, I'm moving it in the right directory...